data/systemd/yggd.service.in

```diff
@@ -1,6 +1,8 @@
 [Unit]
 Description=@BRANDNAME@ daemon
 Documentation=https://github.com/redhatinsights/yggdrasil
+After=network.target network-online.target
```

Comment:
I think we can drop `network.target`. Rereading the documentation on targets[1], `network.target` doesn't sound like what we really need to depend on.

1: https://www.freedesktop.org/software/systemd/man/systemd.special.html#network-online.target